Library/Homebrew/cmd/style.rb

```diff
@@ -23,9 +19,11 @@
 module Homebrew
   def style
     target = if ARGV.named.empty?
-      [HOMEBREW_LIBRARY]
+      [HOMEBREW_LIBRARY/"Homebrew"]
     elsif ARGV.named.any? { |file| File.exist? file }
       ARGV.named
+    elsif ARGV.named.any? { |tap| tap.include?("/") }
```

Comment:
Can you make this check for exactly one `/` or possibly even if all arguments are valid tap names? Otherwise, formula names, that are handled below and can also contain slashes, will be interpreted as taps which will fail.